assets/Product.wxs

```diff
@@ -84,17 +84,17 @@
         </DirectorySearch>
       </DirectorySearch>
     </Property>
-    <Condition Message="$(env.ProductName) requires the Universal C Runtime to be installed. You can find a download link to it here: https://github.com/PowerShell/PowerShell/blob/master/docs/installation/windows.md#prerequisites">
+    <Condition Message="$(env.ProductName) requires the Universal C Runtime to be installed to enable remoting over WinRM. You can find a download link to it here: http://aka.ms/pscore6-prereq">
       <![CDATA[Installed OR UNIVERSAL_C_RUNTIME_INSTALLED]]>
     </Condition>
-    <!-- Prerequisite check for Visual Studio 2015 C++ redistributables -->
-    <Property Id="VISUAL_CPP_RUNTIME_INSTALLED" Secure="yes">
+    <!-- Prerequisite check for Windows Management Framework -->
+    <Property Id="PWRSHPLUGIN_VERSION" Secure="yes">
       <DirectorySearchRef Id="System32" Parent="WindowsDirectory" Path="System32">
-        <FileSearch Id="vcruntime140" Name="vcruntime140.dll"/>
+        <FileSearch Id="pwrshplugin" Name="pwrshplugin.dll" MinVersion="6.3.9600.16383"/>
       </DirectorySearchRef>
     </Property>
-    <Condition Message="$(env.ProductName) requires the Visual Studio 2015 C++ redistributables to be installed. You can find a download link to it here: https://github.com/PowerShell/PowerShell/blob/master/docs/installation/windows.md#prerequisites">
-      <![CDATA[Installed OR VISUAL_CPP_RUNTIME_INSTALLED]]>
+    <Condition Message="$(env.ProductName) requires the Windows Management Framework 4.0 or newer to be installed to enable remoting over WinRM. You can find download links here: http://aka.ms/pscore6-prereq">
```

Comment:
I'm wondering if this check should be a notification during installation rather than a requirement. There is a strong desire from a few folks to have PowerShell Core installable on Win 7 SP1 without requiring a WMF installation. The point is to make PowerShell Core usable on those systems without requiring a reboot. 

The open questions I have on this topic are:
1. To what extent does PowerShell Core function on Win 7 SP1 without WMF 4.0+?
1. Does outgoing remoting function without WMF 4.0 and UCRT?

Reply:
I think if people want to use powershell regardless of the prerequisites, then the `zip` package is the way to go -- no precheck. For `msi` package, it might be better to ensure everything works.

Those two are good open questions.
For UCRT, not sure if WMI/MI.NET would also depend on it.
For WMF 4.0, it's definitely required for making powershell core connectable as server side, but need to see if outgoing remoting needs it on win7.

Reply:
I'm on the side that the MSI should be comprehensive and the zip for advanced deployments (users knows what they are doing).  I think if we get a bunch of customer feedback on the pre-req, we can revisit this.

Reply:
> For UCRT, not sure if WMI/MI.NET would also depend on it.

I tried on Win8.1 and Win2012R2 that don't have UCRT installed, CIM cmdlets seem working fine. So it looks to me that UCRT is only required by `pwrshplugin.dll`.

Reply:
OK, thanks for checking. If outgoing remoting works on Win 7 SP1 without WMF 4.0, then my concerns are covered.

Reply:
> Does outgoing remoting function without WMF 4.0 and UCRT? 

UCRT is not required for outgoing remoting connection. I'm able to create a session in PowerShell core from a Win81 box without UCRT to another Win81 using the configuration created on the latter Win81.

However, it seems WMF 4 is required for outgoing connection. PowerShell core crashed when I tried to create a session from a Win7 box with PSv2 to a Win7 with WMF 5.1 using the configuration created on the latter Win7:

![image](https://user-images.githubusercontent.com/127450/31369279-2e99682e-ad38-11e7-96d4-6f20f7416402.png)